MobileCenter.podspec

```diff
@@ -47,10 +47,10 @@ Pod::Spec.new do |s|
       ss.vendored_frameworks = "MobileCenter-SDK-iOS/MobileCenterCrashes.framework"
   end
 
-#  s.subspec 'MobileCenterUpdates' do |ss|
+#  s.subspec 'MobileCenterDistribute' do |ss|
 #   ss.frameworks = 'CoreTelephony', 'Foundation'
 #   ss.dependency 'MobileCenter/MobileCenter'
-#  ss.resource_bundle = { 'MobileCenterUpdatesResources' => ['MobileCenter-SDK-iOS/MobileCenterUpdatesResources.bundle/*.lproj'] }
+#  ss.resource_bundle = { 'MobileCenterDistributeResources' => ['MobileCenter-SDK-iOS/MobileCenterDistributeResources.bundle/*.lproj'] }
 # end
 
 end
```

Comment:
It will be determined when we merge to develop. The version doesn't mean anything in this stage.